Panel.php

```diff
@@ -49,7 +49,10 @@ class Panel extends Component
      * See [[\yii\base\Controller::actions()]] for the format.
      */
     public $actions = [];
-
+    /**
+     * @var \Exception|null Errors while saving the panel
+     */
+    protected $error;
```

Comment:
Property name is `$error` but description says "errors". I think description should be fixed.